src/modules/userMethods.ts

```diff
@@ -185,6 +186,17 @@ export const userMethods = {
         });
     },
 
+    getByLogin: async (
+        login: string,
+        sessionUser: SessionUser,
+    ): Promise<User & UserMemberships & UserSupervisor & UserAchievements & UserSupervisorOf & UserSupervisorIn> => {
+        const user = await prisma.user.findUnique({ where: { login } });
+
+        if (!user) throw new TRPCError({ code: 'NOT_FOUND', message: `No user with login ${login}` });
+
+        return userMethods.getById(user.id, sessionUser);
```

Comment:
why would you query second time? you already got the user

Reply:
i get only id, and use request `getById` to get the same user with needed includes and schema